[GRAFT][release/r10.18.0] Merge pull request #1031 from SoftwareAG/feature/MTM-50214/MTM-53145-documentation-improvements (#1059)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/managing-permissions.md

To assign inventory roles, click **Users** in the **Accounts** menu, select a user in the user list and switch to its **Inventory roles** tab.

> In the **Inventory roles** tab you will see a tree of device groups. To assign an inventory role, open the dropdown on the right side of the group row. Select the relevant roles and click **Apply**. For a detailed description of a role click the info icon next to it or refer to [Inventory roles](#inventory).

Reviewer: ```suggestion
In the **Inventory roles** tab you will see a tree of device groups. To assign an inventory role, open the dropdown at the right of the group row. Select the relevant roles and click **Apply**. For a detailed description of a role click the info icon next to it or refer to [Inventory roles](#inventory).
```
